lib/stripe.js

```diff
@@ -299,12 +302,19 @@ Stripe.prototype = {
     return formatted;
   },
 
+  setTelemetryEnabled: function(enableTelemetry) {
+    this._enableTelemetry = enableTelemetry;
+  },
+
+  telemetryEnabled: function() {
```

Comment:
It looks like the convention in the rest of this file is to use get/set prefixes on the function names, so I'd change this to `getTelemetryEnabled`.